package.json

```diff
@@ -106,6 +106,17 @@
           "default": true,
           "description": "Should the server log to client output"
         },
+        "xml.format.formatEmptyElements": {
```

Comment:
xml.format.emptyElements

there's already "format" in the key

Reply:
fixed in this PR and fixed on LemMinx side